ansible/roles/oracle-19c/tasks/install_grid.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: Check oracle 19c Grid Infrastructure installed
+  ansible.builtin.stat:
+    path: "{{ oracle_inventory }}/orainstRoot.sh"
+  register: grid_software_installed
```

Comment:
i think we need a better mechanism to see if the passwords are already made before we recreate each one in the grid response file

Reply:
With new install we don't want to use existing passwords. Every time with new install , it should generate the password.